pkg/cloudprovider/aws/aws.go

```diff
@@ -41,6 +49,32 @@ type EC2 interface {
 
 	// Query the EC2 metadata service (used to discover instance-id etc)
 	GetMetaData(key string) ([]byte, error)
+
+	// TODO: It is weird that these take a region.  I suspect it won't work cross-region anwyay.
+	// TODO: Refactor to use a load balancer object?
+	// List load balancers
+	DescribeLoadBalancers(region string, name string) (map[string]elb.LoadBalancer, error)
+	// Create load balancer
```

Comment:
Good point.  It's the k8s name, because elb.LoadBalancer doesn't have a place for the k8s name.

I'll add a comment.